xqueue_watcher/manager.py

```diff
@@ -19,30 +19,53 @@
     'python2': codejail.languages.python2,
     'python3': codejail.languages.python3,
 }
+MANAGER_CONFIG_DEFAULTS = {
+    'HTTP_BASIC_AUTH': None,
+    'POLL_TIME': 10,
+    'REQUESTS_TIMEOUT': 1,
+    'POLL_INTERVAL': 1,
+    'LOGIN_POLL_INTERVAL': 5,
+}
+
+
+def get_manager_config_values(app_config_path):
```

Comment:
Please pull this method and MANAGER_CONFIG_DEFAULTS in to a module named settings.